### ISSUE TITLE
No boxes to NMS

### ISSUE THREAD

**Issue by SaiAinala3** [toandaominh1997/EfficientDet.Pytorch#143]
Hi,

I have trained using : 
python3 train.py --dataset COCO --dataset_root data/coco/ --network efficientdet-d0 --batch_size 8  --num_epoch 15 --num_class 1 --save_folder saved/

After 5 epochs:

> 
>     cls_loss       : 2.9468758384609828e-06
>     reg_loss       : 0.0
>     mean_loss      : 2.9462864633387655e-06
>     time           : 474.67488193511963
>     loss           : 2.9462864633387655e-06
> 
> 15 epoch: 	 start validation....
> No boxes to NMS
> No boxes to NMS
> No boxes to NMS

Even if I run:
python3 eval.py  --dataset COCO --dataset_root data/coco --weight saved3/COCO/efficientdet-d0/checkpoint_14.pth

Output :

> Loading checkpoint: saved3/COCO/efficientdet-d0/checkpoint_14.pth ...
> Loaded pretrained weights for efficientnet-b0
> loading annotations into memory...
> Done (t=0.03s)
> creating index...
> index created!
> No boxes to NMS
> No boxes to NMS
> No boxes to NMS

Can someone help me with this? Thank you.
